backend/api/migrations/0023_auto_20231020_1755.py

```diff
@@ -0,0 +1,31 @@
+# Generated by Django 3.2.16 on 2023-10-20 17:55
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("api", "0022_annotationcomment"),
+    ]
+
+    operations = [
+        migrations.AlterModelOptions(
+            name="annotationcampaign",
+            options={"ordering": ["name", "created_at"]},
+        ),
+        migrations.AlterModelOptions(
+            name="dataset",
+            options={"ordering": ["name", "created_at"]},
+        ),
+        migrations.AddField(
+            model_name="annotationcampaign",
+            name="created_at",
+            field=models.DateTimeField(blank=True, null=True),
+        ),
+        migrations.AddField(
+            model_name="dataset",
+            name="created_at",
+            field=models.DateTimeField(blank=True, null=True),
```

Comment:
I think there's something wrong with this migration.
For this PR, we can see that GitHub actions for backend fail on `if [ "$(poetry run ./manage.py makemigrations --dry-run)" != "No changes detected" ]; then exit 1; fi`

When I try to run `poetry run ./manage.py makemigrations --dry-run` there is indeed a migration to be created. My guess is that current migration has been created before some changes in the PR.

I'm not familiar with Django migrations, so I can't help much. But if I had to repair it, here is what I would have tried:

* Empty database
* Delete migration file `0023_auto_20231020_1755.py`
* Migrate (`poetry run ./manage.py migrate`)
* Recreation migration (`poetry run ./manage.py makemigrations`)

Reply:
@Qouagga either follow or at minimal you should rebase on master and rename to `0024_add_created_at_fields.py` (+ update the dependencies) :wink:

Also you'll need to fix the poetry.lock conflict (ask me if you have any difficulties with this).